UI/forms/OBSBasicTransform.ui

```diff
@@ -15,204 +15,157 @@
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0">
    <item>
-    <layout class="QFormLayout" name="formLayout">
-     <property name="fieldGrowthPolicy">
-      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
-     </property>
-     <property name="labelAlignment">
-      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
-     </property>
-     <item row="0" column="0">
-      <widget class="QLabel" name="label">
-       <property name="sizePolicy">
-        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
-         <horstretch>0</horstretch>
-         <verstretch>0</verstretch>
-        </sizepolicy>
-       </property>
-       <property name="minimumSize">
-        <size>
-         <width>170</width>
-         <height>0</height>
-        </size>
-       </property>
-       <property name="accessibleName">
-        <string>Basic.TransformWindow.Position</string>
-       </property>
-       <property name="text">
-        <string>Basic.TransformWindow.Position</string>
-       </property>
-       <property name="alignment">
-        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
-       </property>
-      </widget>
-     </item>
-     <item row="0" column="1">
-      <widget class="QFrame" name="widget">
-       <property name="sizePolicy">
-        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
-         <horstretch>0</horstretch>
-         <verstretch>0</verstretch>
-        </sizepolicy>
-       </property>
-       <layout class="QHBoxLayout" name="horizontalLayout">
-        <property name="leftMargin">
-         <number>0</number>
-        </property>
-        <property name="topMargin">
-         <number>0</number>
-        </property>
-        <property name="rightMargin">
-         <number>0</number>
-        </property>
-        <property name="bottomMargin">
-         <number>0</number>
-        </property>
-        <item>
-         <widget class="QDoubleSpinBox" name="positionX">
+    <widget class="QWidget" name="container" native="true">
+     <layout class="QGridLayout" name="gridLayout_2">
```

Comment:
Apologies for only seeing this now, but is this additional grid layout intentional?
I hope "Hide Whitespaces" isn't deceiving me here, but it looks like only (0, 0) of the layout gets filled with an item (the form layout), in which case I think the grid layout would be redundant? Hope I'm not missing something in the diff here.

Reply:
A widget needs to have a layout to have other widgets inside of it.

Reply:
Sure, but - as far as I understand - here we have

`container` (widget)
\- `gridLayout_2` (layout)
\- \- [0, 0] `formLayout` (layout)

So gridLayout seems redundant, doesn't it?

Reply:
Updated to remove the extra layout.